platforms/chibios/boards/QMK_KB2040/configs/config.h

```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#ifndef I2C_DRIVER
+#    define I2C_DRIVER I2CD2
+#endif
+#ifndef I2C1_SDA_PIN
+#    define I2C1_SDA_PIN D2
+#endif
+#ifndef I2C1_SCL_PIN
+#    define I2C1_SCL_PIN D3
+#endif
```

Comment:
```suggestion
#endif

#ifndef RP2040_BOOTLOADER_DOUBLE_TAP_RESET
#    define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
#endif
#ifndef RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED
#    define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP17
#endif
#ifndef RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT
#    define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 1000U
#endif
```

Are these appropriate defaults that can be set here for the controller?

Reply:
`RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED` isnt super happy after reset as nothing then turns the led off....

Reply:
It might be possible with `gpio_init(17U)` and `gpio_put(17U)` from `pico/stdlib.h`.

Reply:
The bootloader led pin isn't meant for smart LEDs, only regular LEDs. This is a bit strange but you can see it in the sources of the pico bootrom. Either we add an de-init sequence for smart LEDs or just don't set it at all.

Reply:
The fact it lights up, when its not intended for smart LEDs is also something i don't think is worth relying on.

For now, i've not set the option and we can re-evaluate at a later date.